pkgs/desktops/gnome-3/update.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, writeScript, python3, common-updater-scripts, coreutils, gnugrep, gnused }:
+{ packageName, attrPath ? packageName, versionPolicy ? "odd-unstable" }:
+
+let
+  python = python3.withPackages (p: [ p.requests ]);
+in writeScript "update-${packageName}" ''
+  set -o errexit
+  PATH=${lib.makeBinPath [ common-updater-scripts coreutils gnugrep gnused python ]}
+  latest_tag=$(python "${./find-latest-version.py}" "${packageName}" "${versionPolicy}" "stable")
```

Comment:
I do not see an easy way to make the stability configurable from `maintainers/scripts/update.nix`. I guess I will just have to change it manually when working on unstable branch.